presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/JtsGeometryUtils.java

```diff
@@ -239,12 +332,63 @@ private static Geometry readPolygon(SliceInput input)
         return GEOMETRY_FACTORY.createMultiPolygon(polygons.toArray(new Polygon[0]));
     }
 
+
+    private static void writePolylineOrPolygon(GeometryCollection geometries,
```

Comment:
I don't think this method works. Consider adding a set of tests similar to the ones in TestJtsGeometryUtils.

Reply:
working on it